services/QuillLMS/client/app/bundles/Shared/utils/stringFormatting.ts

```diff
@@ -0,0 +1,15 @@
+
+class StringFormatting {
+  static highlightSpellingGrammar = (str: string, wordsToFormat: string | string[]) => {
+    let wordArray = [].concat(wordsToFormat)
+    let newString = str
+    wordArray.forEach((word) => {
+      var matched = newString.match(`${word}[\\w\']*`)
+      var augmentedRegex = new RegExp(`${matched[0]}`, 'g')
```

Comment:
Use `const` instead of `var` here: https://airbnb.io/javascript/#references--prefer-const